code/mob/living/critter/gunbot.dm

```diff
@@ -18,6 +18,12 @@
 	speechverb_ask = "queries"
 	metabolizes = 0
 
+	New()
+		. = ..()
+		var/image/eye_light = image(icon, "mars_sec_bot_eye")
+		eye_light.plane = PLANE_SELFILLUM
+		src.UpdateOverlays(eye_light, "eye_light")
```

Comment:
so you're saying this doesn't work?

Reply:
Oh no that works, it's just pretty different to the implementation on borgs which use black pixels for eyes and put them on lighting plane.

https://github.com/goonstation/goonstation/blob/697eb5d04de29942d325694dcba639db181a12b9/code/mob/living/silicon/robot.dm#L2590